opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -1268,6 +1283,11 @@ def _create_aggregation(
             )
 
         if isinstance(instrument, Histogram):
+            boundaries: Optional[Sequence[float]] = (
+                instrument._advisory.explicit_bucket_boundaries
+                if instrument._advisory.explicit_bucket_boundaries is not None
+                else None
+            )
```

Comment:
nit I think this is equivalent
```suggestion
            boundaries = instrument._advisory.explicit_bucket_boundaries
```